### DISCUSSION TITLE
A question get lost in discord to snowswap

### DISCUSSION THREAD

**Discussion by hackfisher** [darwinia-network/darwinia#1575] [Q&A] [answered]
> @woshijiucai Can you verify that the previous Snowswap rewards receiving address used for the KTON/RING Farm is still the same?
> Thank you very much :+1:
> 0xF7a46a0932708eA27E4b2de7283b94Cb553523b6
> https://darwinia.subscan.io/account/0xF7a46a0932708eA27E4b2de7283b94Cb553523b6?tab=contract

**Comment by hackfisher** [comment]
@WoeOm

**Reply by WoeOm** [reply]
The stake-starter contract cannot be reused. The farm only supports distribute a fixed number of tokens within a fixed period of time, and does not support repeated additional rewards.

**Reply by 2xJDubs** [reply]
> The stake-starter contract cannot be reused. The farm only supports distribute a fixed number of tokens within a fixed period of time, and does not support repeated additional rewards.

Thank you for the clarification.  Passed on this a new farm pool will need to be created to keep pool lp incentives going?

I have also noticed that more than half of the reward pool is left but we are well past half way of the 6month period the pool was to last.  I think the distribution calculations are off for the farm.  Recently, in the last few days, rewards distribution has significantly decreased as well.

**Reply by WoeOm** [reply, marked as answer]
Synchronize the info. On 25 Auguest, We have communicated with Partnership Dude about farm-related issues.

1. There are a lot of RING that have not been claimed. The reason is that many stakers have not claimed their rewards.
2. Receiving contract: 0x6b8D7682a5ed1063EdE11E7fB74Fd52b451BD0c2

https://explorer.darwinia.network/address/0x6b8D7682a5ed1063EdE11E7fB74Fd52b451BD0c2?tab=contract

start time 2024-10-10 00:00 UTC
reward amount 1,400,000 RING
rewards Token 0xE7578598Aac020abFB918f33A20faD5B71d670b4
ring-kton lp 0x28B4579C37E82D6F57F88d35D34591E7Cd26398E 
duration 15552000
